otter/test/worker/test_launch_server_v1.py

```diff
@@ -2423,6 +2421,9 @@ def test_delete_and_verify_limits(self):
         :func:`delete_and_verify` limits the number of delete server requests
         to 1. It also delays response by 1 second
         """
+        set_config_data({"worker": {"delete_server_limit": 1}})
+        self.addCleanup(set_config_data, None)
```

Comment:
Should all calls to `set_config_data` work this way?  Maybe there should be a `set_config_data_for_test` function?